### PR TITLE
added comparison to binance smart chain

### LINKED ISSUE #1555: Add comparison to Binance Smart Chain
I know it's quite different, but I've gotten three different people asking me what the difference is between Polkadot and BSC.

### PR DESCRIPTION
**Pull request by midegdugarova**
fixes #1555

### CODE REVIEW

**Comment by Swader** [comment]
Is this something people are curious about? Seems a bit superfluous to me.

**Comment by laboon** [comment]
I've gotten quite a few questions about BSC vs Polkadot. I think it makes sense to add it.

File: docs/learn-comparisons.md [hunk 1]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+
+Polkadot has an entirely different purpose because it was built to connect and secure unique
+blockchains. It is a protocol on which single blockchains (such as Binance Smart Chain) could be
+built and benefit from shared security, interoperability and scalability. Interoperability within
+Polkadot is based on pooled security on Polkadot, and the security of the entire Polkadot network,
+and has much stronger trust assumptions.
+
+Scalability based on bridges relies on each bridged chain finding its own set of validators
+therefore duplicate resources are required. Scalability on Polkadot is based on the security of the
+Relay chain, and as more validators join Polkadot, more parachains can be supported.
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
Relay Chain, and as the number of validators in the active set on Polkadot are increased, more parachains can be supported.
```

File: docs/learn-comparisons.md [hunk 2]
```diff
@@ -39,3 +39,27 @@ Chain.
 
 As such, Polkadot is a possible augmentation and scaling method for Ethereum 1.x, rather than
 competition.
+
+## Binance Smart Chain
+
+[Binance Chain](https://www.binance.com) is a Proof of Staked Authority blockchain is used to
```
Comment (Swader) [inline docs/learn-comparisons.md]:
```suggestion
[Binance Chain](https://www.binance.com) is a Proof of Stake Authority blockchain used to
```

Comment (laboon) [inline docs/learn-comparisons.md:45]:
```suggestion
[Binance Chain](https://www.binance.com) is a Proof of Staked Authority blockchain used to
```

File: docs/learn-comparisons.md [hunk 3]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+exchange digital assets on the blockchain (Binance DEX). Binance Smart Chain is an
+Ethereum-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
+Dual Chain System. Binance Smart Chain is also a Proof of Staked Authority chain and allows users to
```
Comment (Swader) [inline docs/learn-comparisons.md]:
```suggestion
Dual Chain System. Binance Smart Chain is also a Proof of Stake Authority chain and allows users to
```

File: docs/learn-comparisons.md [hunk 4]
```diff
@@ -39,3 +39,27 @@ Chain.
 
 As such, Polkadot is a possible augmentation and scaling method for Ethereum 1.x, rather than
 competition.
+
+## Binance Smart Chain
+
+[Binance Chain](https://www.binance.com) is a Proof of Staked Authority blockchain is used to
+exchange digital assets on the blockchain (Binance DEX). Binance Smart Chain is an
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
exchange digital assets on Binance DEX. Binance Smart Chain is an
```

File: docs/learn-comparisons.md [hunk 5]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+Ethereum-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
+Dual Chain System. Binance Smart Chain is also a Proof of Staked Authority chain and allows users to
+create smart contracts and dApps.
+
+Both chains are built with Cosmos SDK and therefore are a part of the
+[Cosmos](https://wiki.polkadot.network/docs/en/learn-comparisons-cosmos) ecosystem. Due to specifics
+of the Cosmos architecture, interoperability of Binance Smart Chain is based on bridges. This means
```
Comment (Swader) [inline docs/learn-comparisons.md:52]: interoperability with what?

File: docs/learn-comparisons.md [hunk 6]
```diff
@@ -39,3 +39,27 @@ Chain.
 
 As such, Polkadot is a possible augmentation and scaling method for Ethereum 1.x, rather than
 competition.
+
+## Binance Smart Chain
+
+[Binance Chain](https://www.binance.com) is a Proof of Staked Authority blockchain is used to
+exchange digital assets on the blockchain (Binance DEX). Binance Smart Chain is an
+Ethereum-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
EVM-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
```

File: docs/learn-comparisons.md [hunk 7]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+Dual Chain System. Binance Smart Chain is also a Proof of Staked Authority chain and allows users to
+create smart contracts and dApps.
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
create smart contracts and dapps.
```

File: docs/learn-comparisons.md [hunk 8]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+[Cosmos](https://wiki.polkadot.network/docs/en/learn-comparisons-cosmos) ecosystem. Due to specifics
+of the Cosmos architecture, interoperability of Binance Smart Chain is based on bridges. This means
+all validators of both chains are also bridge operators, therewith the security of the system relies
+on trusting validators. At the moment there are 21 Binance Smart Chain validator nodes.
+
+Polkadot has an entirely different purpose because it was built to connect and secure unique
+blockchains. It is a protocol on which single blockchains (such as Binance Smart Chain) could be
+built and benefit from shared security, interoperability and scalability. Interoperability within
+Polkadot is based on pooled security on Polkadot, and the security of the entire Polkadot network,
+and has much stronger trust assumptions.
```
Comment (Swader) [inline docs/learn-comparisons.md]:
```suggestion
and has much stronger economic security.
```

File: docs/learn-comparisons.md [hunk 9]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+[Binance Chain](https://www.binance.com) is a Proof of Staked Authority blockchain is used to
+exchange digital assets on the blockchain (Binance DEX). Binance Smart Chain is an
+Ethereum-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
+Dual Chain System. Binance Smart Chain is also a Proof of Staked Authority chain and allows users to
+create smart contracts and dApps.
+
+Both chains are built with Cosmos SDK and therefore are a part of the
+[Cosmos](https://wiki.polkadot.network/docs/en/learn-comparisons-cosmos) ecosystem. Due to specifics
+of the Cosmos architecture, interoperability of Binance Smart Chain is based on bridges. This means
+all validators of both chains are also bridge operators, therewith the security of the system relies
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
all validators of both chains are also bridge operators, therefore the security of the system relies
```

File: docs/learn-comparisons.md [hunk 10]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+on trusting validators. At the moment there are 21 Binance Smart Chain validator nodes.
+
+Polkadot has an entirely different purpose because it was built to connect and secure unique
+blockchains. It is a protocol on which single blockchains (such as Binance Smart Chain) could be
+built and benefit from shared security, interoperability and scalability. Interoperability within
+Polkadot is based on pooled security on Polkadot, and the security of the entire Polkadot network,
+and has much stronger trust assumptions.
+
+Scalability based on bridges relies on each bridged chain finding its own set of validators
```
Comment (Swader) [inline docs/learn-comparisons.md]:
```suggestion
Scalability based on bridges relies on each bridged chain finding its own set of validators,
```

File: docs/learn-comparisons.md [hunk 11]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+exchange digital assets on the blockchain (Binance DEX). Binance Smart Chain is an
+Ethereum-compatible smart contract chain bridged to Binance Chain. Together they form the Binance
+Dual Chain System. Binance Smart Chain is also a Proof of Staked Authority chain and allows users to
+create smart contracts and dApps.
+
+Both chains are built with Cosmos SDK and therefore are a part of the
+[Cosmos](https://wiki.polkadot.network/docs/en/learn-comparisons-cosmos) ecosystem. Due to specifics
+of the Cosmos architecture, interoperability of Binance Smart Chain is based on bridges. This means
+all validators of both chains are also bridge operators, therewith the security of the system relies
+on trusting validators. At the moment there are 21 Binance Smart Chain validator nodes.
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
on trusting validators. At the moment, there are 21 Binance Smart Chain validator nodes.
```

**Review by Swader — CHANGES_REQUESTED** [review]

File: docs/learn-comparisons.md [hunk 12]
```diff
@@ -39,3 +39,27 @@ Chain.
...
+
+Polkadot has an entirely different purpose because it was built to connect and secure unique
```
Comment (laboon) [inline docs/learn-comparisons.md]:
```suggestion
Polkadot has an entirely different purpose, as it was built to connect and secure unique
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Review by Swader — APPROVED** [review]

**Comment by Swader** [comment]
Looks OK to me, though I still fail to find a tangent between the two chains (to me this is like comparing MySQL and HTML) so not sure if we should point out that they have absolutely nothing in common, from mission to execution. I'll defer to @laboon for the final verdict here.

**Comment by laboon** [comment]
I mean, it's basically a PoA'd Eth1 and we have a comparison to that... to many people a blockchain is a blockchain. I wouldn't recommend including comparisons to every chain out there, of course, but we get enough questions about it that I think it's worth it so we can just send a link to this instead of repeating ourselves.

**Pull request merged**